### PR TITLE
Visual Revamp: First pass at colors and tabs

### PR DESCRIPTION
**Pull request by wassgha**
## Summary

Screen Shot 2020-08-03 at 1 20 31 AM

First pass at the implementation of @samitron7's new [Ueno Visual Design](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Sam---Visual-Design-Frames?node-id=0%3A1) starting with color scheme changes and the improved tab design.

## Relevant Technical Choices

- Major refactor to the library pane and inspector pane tabs in order to use a new common `TabView` component. 
- Changed color names to match the new design language
- Added storybook example for the `TabView` component

## User-facing changes

- Dark mode colors partially implemented
- Tabs now adopt the new design (on the library and inspector panels)
- Minor changes including rounded borders on the masonry and page previews

## To-do
- Remove legacy colors
- Continue implementing the design in future PRs (moving buttons around, new masonry design, background media changes, etc.)

## Testing Instructions

- Visual changes should be present on the editor as well as on Storybook

---

Partial for #3641

### CODE REVIEW

File: assets/src/edit-story/components/moveable/cropStyle.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ export const GlobalStyle = createGlobalStyle`
 	}
 
 	.crop-moveable .moveable-direction.moveable-line {
-		background: #47a0f4 !important;
+		background: #1A73E8 !important;
```
Comment (dvoytenko) [inline assets/src/edit-story/components/moveable/cropStyle.js]: No way to bring themes here? If not, can we at least put in a comment `== theme.x.y.z`?

File: assets/src/edit-story/style.css [hunk 2]
```diff
@@ -32,7 +32,7 @@ body.edit-story #wpbody-content {
 }
 
 body.edit-story #edit-story {
-  background-color: #191c28;
+  background-color: #1b1d1c;
```
Comment (dvoytenko) [inline assets/src/edit-story/style.css]: Ditto: reference the theme notation.

**Review by dvoytenko — APPROVED** [review]
Approved with two asks.

File: assets/src/edit-story/components/canvas/pagepreview/index.js [hunk 3]
```diff
@@ -68,6 +68,7 @@ const PreviewWrapper = styled.div`
   position: relative;
   overflow: hidden;
   background-color: white;
+  border-radius: 4.5px;
```
Comment (swissspidy) [inline assets/src/edit-story/components/canvas/pagepreview/index.js:71]: Will we be using the same radius more often? Might be worth putting it into the theme then.

Comment (wassgha) [reply assets/src/edit-story/components/canvas/pagepreview/index.js:71]: Doesn't look like it, this is specific to the small preview

**Comment by wassgha** [comment]
@swissspidy any idea why the PHP check is failing?

**Comment by swissspidy** [comment]
@wassgha PHP 8.0 check is currently failing but you can ignore it. It is marked as allowed-to-fail and does not block merging PRs. There's no "required" mark next to it.

**Comment by swissspidy** [comment]
The Percy snapshots need review though, that's what's currently blocking this PR.

Screenshot 2020-08-04 at 10 44 26

**Comment by wassgha** [comment]
> The Percy snapshots need review though, that's what's currently blocking this PR.
> 
> 
> 
> Screenshot 2020-08-04 at 10 44 26
> 
> 

Looks like I don't have access to approve Percy :/

**Comment by swissspidy** [comment]
@wassgha Invite sent

**Comment by samitron7** [comment]
@wassgha There are still areas where we need colors to be changed

**Layers panel**
Can we use color value of "BG Divider" for this.
Screen Shot 2020-08-05 at 7 12 40 AM

**Video settings toggle in the Design Panel**
Screen Shot 2020-08-05 at 7 11 17 AM

**Required text and Page advancement toggle in the Document Panel**
Color value to "required" should be "secondary" and not blue. Let's make any links in the panels have a color value of "biru" 
Screen Shot 2020-08-05 at 7 11 33 AM

**Pull request merged**